docs/docs/components/view.md

```diff
@@ -79,6 +79,10 @@ restrictFocusWithin: boolean = false;
 // changing it during the View life cycle will produce an error.
 limitFocusWithin: LimitFocusType = LimitFocusType.Unlimited;
 
+// Should be focused when the component is mounted,
+// see https://microsoft.github.io/reactxp/docs/apis/focusutils.html
+autoFocus: AutoFocusValue = undefined;
```

Comment:
>autoFocus [](start = 0, length = 9)

looks like this should trigger focusable rendering for UWP. I'll add Radu who owns that.